static/app/views/performance/trends/index.tsx

```diff
@@ -66,13 +66,14 @@ class TrendsSummary extends Component<Props, State> {
   };
 
   renderContent() {
-    const {organization, location} = this.props;
+    const {organization, location, projects} = this.props;
     const {eventView} = this.state;
     return (
       <TrendsContent
         organization={organization}
         location={location}
         eventView={eventView}
+        projects={projects}
```

Comment:
@udameli would you be able to add a test in https://github.com/getsentry/sentry/blob/a0cac1906303a5e097c479dde03120f46d9ff83b/tests/js/spec/views/performance/trends/index.spec.jsx ?

Reply:
thanks! 🙏